server/form-pages/apply/area-and-funding/funding-information/alternativeID.ts

```diff
@@ -0,0 +1,137 @@
+import errorLookups from '../../../../i18n/en/errors.json'
+import { nameOrPlaceholderCopy } from '../../../../utils/utils'
+import { getQuestions } from '../../../utils/questions'
+
+const applicationQuestions = getQuestions('')['funding-information']['alternative-identification']
+
+const alternativeIDOptions = applicationQuestions.alternativeIDDocuments.answers
+
+export type AlternativeIdentificationBody = {
+  alternativeIDDocuments: Array<keyof typeof alternativeIDOptions>
+  other: string
+}
+
+@Page({
+  name: 'alternative-identification',
+  bodyProperties: ['alternativeIDDocuments', 'other'],
+})
+export default class AlternativeIdentification implements TaskListPage {
+  documentTitle = 'What identification documentation (ID) does the person have?'
+
+  title
+
+  personName = nameOrPlaceholderCopy(this.application.person)
+
+  questions
+
+  body: AlternativeIdentificationBody
+
+  guidanceHtml = `The applicant needs ID if they are applying for Universal Credit for financial support, and Housing Benefit to cover their rent.<br /><br />If they want to receive an advance payment of Universal Credit on the day of release, they will need a bank account and photo ID.`
+
+  constructor(
+    body: Partial<AlternativeIdentificationBody>,
+    private readonly application: Application,
+  ) {
+    this.questions = getQuestions(this.personName)['funding-information']['alternative-identification']
+    this.title = this.questions.alternativeIDDocuments.question
+  }
+
+  previous() {
+    return 'identification'
+  }
+
+  next() {
+    return 'national-insurance'
+  }
+
+  errors() {
+    const errors: TaskListErrors<this> = {}
+    if (!this.body.alternativeIDDocuments) {
+      errors.alternativeIDDocuments = errorLookups.alternativeIDDocuments.empty
+    } else if (this.body.alternativeIDDocuments.includes('other') && !this.body.other) {
+      errors.other = errorLookups.alternativeIDDocuments.other
+    }
+    return errors
+  }
+
+  items(conditionalHtml: string) {
+    const workAndEmploymentOptions = (({ contract, tradeUnion, invoice, hmrc }) => ({
+      contract,
+      tradeUnion,
+      invoice,
+      hmrc,
+    }))(alternativeIDOptions)
+
+    const citizenshipOptions = (({
+      citizenCard,
+      foreignBirthCertificate,
+      citizenCertificate,
+      residenceCard,
+      residencePermit,
+      biometricResidencePermit,
+      laRentCard,
+    }) => ({
+      citizenCard,
+      foreignBirthCertificate,
+      citizenCertificate,
+      residenceCard,
+      residencePermit,
+      biometricResidencePermit,
+      laRentCard,
+    }))(alternativeIDOptions)
+
+    const marriageOptions = (({ marriageCertificate, divorcePapers, dissolutionPapers }) => ({
+      marriageCertificate,
+      divorcePapers,
+      dissolutionPapers,
+    }))(alternativeIDOptions)
+
+    const financialOptions = (({ buildingSociety, councilTax, insurance, chequeBook, mortgage, savingAccount }) => ({
```

Comment:
wow so javascripty! Is this just an alternative to separating the `answers` by keys?, e.g. 
```
answers: {
 workAndEmploymentOptions: {
            contract: 'Employer letter/contract of employment',
            tradeUnion: 'Trade union membership card',
            invoice: 'Invoices (self-employed)',
            hmrc: 'HMRC correspondence',
            }
       }
```

Reply:
haha I know! yes it is, I thought that given `workAndEmploymentOptions` are not really a concept in our data, this approach was preferable to modifying the `questions` data structure and using a custom `response` method.